presto-raptor/src/test/java/com/facebook/presto/raptor/systemtables/TestShardMetadataRecordCursor.java

```diff
@@ -84,6 +96,27 @@ public void setup()
                 .column("orderdate", DATE)
                 .property("temporal_column", "orderdate")
                 .build());
+        metadata.createTable(SESSION, tableMetadataBuilder(TIMESTAMP_TABLE)
+                .column("timestamp", TIMESTAMP)
+                .property("temporal_column", "timestamp")
+                .build());
+    }
+
+    private List<ShardInfo> createTestShards(List<List<ColumnStats>> shardColumnStats)
```

Comment:
I think this can be static. Also, put helper methods at the bottom of the class.